src/components/Pages/TeamMembers/Add/Add.js

```diff
@@ -51,11 +50,23 @@ function Add(props) {
     );
   };
 
+  const { teamMember: teamMemberState } = state;
+
+  useEffect(() => {
+    // Checks input conditions.  If all required field conditions are met, Add Member button is activated
+    const { first_name, last_name, job_description } = teamMemberState;
+    //console.log(first_name.length, )
+    const payload = !(
+      first_name.length >= 3 &&
+      last_name.length >= 3 &&
+      job_description.length >= 3 &&
```

Comment:
@ajb85 I actually already removed the length restrictions on these fields in the BE Joi validation, this can be removed